contracts/OBSSStorage.sol

```diff
@@ -98,6 +98,28 @@ contract OBSSStorage is OwnableUpgradeable, ERC2771Recipient {
     feeds = Feeds(_feeds);
   }
 
+  // Setters
```

Comment:
Should we make 1 function that combines all setters? Maybe `upgradeInitData`?

Reply:
This feels a bit non-standard for me for Solidity, but I'm not too familiar.